avocado/utils/service.py

```diff
@@ -118,8 +118,9 @@ def method(cmd_result):
             # If service is stopped, exit_status is also not zero.
             # So, we can't use exit_status to check result.
             output = cmd_result.stdout.lower()
+            error = cmd_result.stderr.lower()
             # Returns None if XXX is unrecognized.
-            if re.search(r"unrecognized", output):
+            if re.search(r"unrecognized", error):
```

Comment:
OK RedHat 6.1 also reports this in `stderr`, I think it's safe to merge this :smile: